trieste/acquisition/function.py

```diff
@@ -26,6 +26,13 @@
 
 AcquisitionFunction = Callable[[QueryPoints], tf.Tensor]
 """ Type alias for acquisition functions. """
+BatchAcquisitionFunction = Callable[[QueryPoints], tf.Tensor]
+""" 
+Type alias for batch acquisition functions. 
+
+While an AcquisitionFunction handles query points of shape [..., D] and returns [..., 1] values,
+BatchAcquisitionFunction handles batches of query points of shape [..., B, D] and returns [..., 1] values.
+"""
```

Comment:
yes, this is what I had in mind. You could even put the bit about `AcquisitionFunction`s in the `AcquisitionFunction` docstring